resin/models/build.py

```diff
@@ -43,9 +43,28 @@ def get(self, id):
         else:
             raise exceptions.BuildNotFound(id)
 
+    def get_by_commit(self, commit_hash):
+        """
+        Get a specific build by commit hash.
+        """
+
+        params = {
+            'filter': 'commit_hash',
+            'eq': commit_hash
+        }
+
+        build = self.base_request.request(
+            'build', 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint')
+        )
+        if build and (len(build['d']) > 0):
```

Comment:
@dfunckt, I've updated the conditions here to make sure it won't return an empty list.